apps/condo/domains/acquiring/utils/taskSchema/index.js

```diff
@@ -58,13 +58,17 @@ async function getAllReadyToPayRecurrentPaymentContexts (context, date, pageSize
     const paymentDayCondition = paymentDay !== endOfMonthPaymentDay
         ? { paymentDay }
         : { paymentDay_gte: paymentDay }
+    const autoPayReceipts = get(extraArgs, 'autoPayReceipts')
```

Comment:
you need to add tests for that case

Reply:
they are already there. I just rewrite it a bit to handle last day of the week case properly